doc/requirements.txt

```diff
@@ -2,4 +2,4 @@ Sphinx==7.2.6
 sphinx_markdown_tables
 myst_parser==2.0.0
 sbom2doc
-pillow>=10.0.1 # not directly required, pinned by Snyk to avoid a vulnerability
\ No newline at end of file
+pillow>=10.3.0 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
```suggestion
pillow>=10.3.0 # not directly required, pinned by Snyk to avoid a vulnerability

```

Adding whitespace final line because every once in a while not having one in a file causes problems.  Also because I need to re-run gitlint with the correct title.